File: BackendDBD/src/main/java/uni/edu/pe/modulo_crm/dto/Mantenimientodto/ListaInfraestructuras.java
package uni.edu.pe.modulo_crm.dto.Mantenimientodto;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ListaInfraestructuras {
    private String idInfraestructura;
    private String nombre;
    private String fechaAdquisicion;
    private String ubicacion;
    private String estado;
    private String tipo;
    private Date ultimoMantenimiento;
    private String proximoMantenimiento;
}

File: FrontendDBD/frontend/src/app/models/modelsmantenimiento/lista-infraestructuras.ts
export interface ListaInfraestructuras {
    idInfraestructura: string;
    nombre: string;
    fechaAdquisicion: Date | null;
    ubicacion: string;
    estado: string;
    tipo: string;
    ultimoMantenimiento: Date | null;
    proximoMantenimiento: Date | null;
}